### PR TITLE
[SYCL] Add test cases for muptiplies,bit_or,bit_xor,bit_and subgroup …

### PR DESCRIPTION
**Pull request by v-klochkov**
…algorithms

Signed-off-by: Vyacheslav N Klochkov <vyacheslav.n.klochkov@intel.com>

### CODE REVIEW

**Comment by v-klochkov** [comment]
The buildbot on Linux reports an issue (SYCL::Plugin/interop-level-zero-keep-ownership.cpp) unrelated to this patch.

File: SYCL/SubGroup/scan_spirv13_fp64.cpp [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+// RUN: %ACC_RUN_PLACEHOLDER %t.out
+
+// This test verifies the correct work of SPIR-V 1.3 exclusive_scan() and
+// inclusive_scan() algoriths used with the MUL operation.
+
+#include "scan.hpp"
+
+int main() {
+  queue Queue;
+  if (!core_sg_supported(Queue.get_device())) {
```
Comment (vladimirlaz) [inline SYCL/SubGroup/scan_spirv13_fp64.cpp]: shouldn't we check that "cl_khr_fp64" extension is supported?

File: SYCL/SubGroup/scan_spirv13_fp16.cpp [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+
+// This test verifies the correct work of SPIR-V 1.3 exclusive_scan() and
+// inclusive_scan() algoriths used with the MUL operation.
+
+#include "scan.hpp"
+
+int main() {
+  queue Queue;
+  if (!core_sg_supported(Queue.get_device())) {
```
Comment (vladimirlaz) [inline SYCL/SubGroup/scan_spirv13_fp16.cpp]: shouldn't we check that "cl_khr_fp16" extension is supported?

**Review by vladimirlaz — CHANGES_REQUESTED** [review]

File: SYCL/SubGroup/scan_spirv13_fp64.cpp (hunk 1 above)
Comment (v-klochkov) [reply SYCL/SubGroup/scan_spirv13_fp64.cpp]: Fixed in 925ce426bb754423531f70b26482e9c98005857a
Thank you!

File: SYCL/SubGroup/scan_spirv13_fp16.cpp (hunk 2 above)
Comment (v-klochkov) [reply SYCL/SubGroup/scan_spirv13_fp16.cpp]: Fixed in 925ce426bb754423531f70b26482e9c98005857a Thank you!

**Review by vladimirlaz — APPROVED** [review]

**Comment by v-klochkov** [comment]
@vladimirlaz - thank you for the approval. The only failed test is on linux buildbot: it is for unrelated test: SYCL::Plugin/interop-level-zero-keep-ownership.cpp

**Pull request merged**